src/lib/db.js

```diff
@@ -207,18 +216,29 @@ function listObservations (cb) {
   }
 }
 
-function observationToFeature (obs, id) {
+function observationToFeature ({ version_id, value }, id) {
   var feature = {
     id,
     type: 'Feature',
     geometry: null,
-    properties: Object.assign({ id }, obs.tags)
+    properties: Object.assign({ id, _version_id: version_id }, value.tags)
   }
-  if (obs.lon && obs.lat) {
+  if (value.lon && value.lat) {
```

Comment:
This should be covered in the [parent function](https://github.com/osmlab/field-data-coordinator/blob/f18de01f9d1a83ec618943d662a586547fad4810/src/lib/db.js#L207). `observationToFeature` is also not used anywhere else, and it feels weird to return `null` or something like that at this stage.